crates/core/executor/src/syscalls/code.rs

```diff
@@ -169,7 +178,9 @@ impl SyscallCode {
             0x00_01_01_29 => SyscallCode::BN254_FP2_ADD,
             0x00_01_01_2A => SyscallCode::BN254_FP2_SUB,
             0x00_01_01_2B => SyscallCode::BN254_FP2_MUL,
-            0x00_00_01_1C => SyscallCode::BLS12381_DECOMPRESS,
```

Comment:
Is this remove intended?

Reply:
oh yes, thats an accident, thanks for catching it!